tests/packagedcode/data/rubygems/gemspec/cocoapods.gemspec.expected.json

```diff
@@ -0,0 +1,216 @@
+      },
+      {
+        "purl": "pkg:gem/cocoapods-search",
+        "requirement": ">= 1.0.0, < 2.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/cocoapods-trunk",
+        "requirement": ">= 1.4.0, < 2.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/cocoapods-try",
+        "requirement": ">= 1.1.0, < 2.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/molinillo",
+        "requirement": "~> 0.6.6",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/xcodeproj",
+        "requirement": ">= 1.14.0, < 2.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/activesupport",
+        "requirement": ">= 4.0.2, < 5",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/colored2",
+        "requirement": "~> 3.1",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/escape",
+        "requirement": "~> 0.0.4",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/fourflusher",
+        "requirement": ">= 2.3.0, < 3.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/gh_inspector",
+        "requirement": "~> 1.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/nap",
+        "requirement": "~> 1.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/ruby-macho",
+        "requirement": "~> 1.4",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/addressable",
+        "requirement": "~> 2.6",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      }
+    ],
+    "contains_source_code": null,
+    "source_packages": [],
+    "purl": "pkg:gem/cocoapods@Pod::VERSION",
```

Comment:
Where does `Pod::VERSION` come from?

Reply:
https://github.com/CocoaPods/CocoaPods/blob/master/cocoapods.gemspec 
They actually used this value in gemspec file.